app/javascript/src/components/ReactVoiceRecorder.css

```diff
@@ -0,0 +1,237 @@
+/* add css module styles here (optional) */
```

Comment:
is this all like generated styles from the library?

Reply:
Initially yes, but I tweaked it a decent bit to make it fit more with the rest of the Storyboard app.